presto-docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -19,7 +19,7 @@ To use Hudi, we need:
 * Network access from the Presto coordinator and workers to the distributed object storage.
 * Access to a Hive metastore service (HMS).
 * Network access from the Presto coordinator to the HMS. Hive metastore access with the Thrift
-protocol defaults to using port 9083.
+  protocol defaults to using port 9083.
```

Comment:
does this fix the docs?
Can you please add a before and after screenshot for the docs in the test plan